### DISCUSSION TITLE
Allow - "No rebasing if you have made edits"

### DISCUSSION THREAD

**Discussion by angelbarrera92** [renovatebot/renovate#28037] [Request Help] [answered]
### What would you like help with?

Other

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

BitBucket - latest

### Please tell us more about your question or problem

Hi!

I've read the docs at https://docs.renovatebot.com/updating-rebasing/#no-rebasing-if-you-have-made-edits

> First of all, here is the one time when Renovate won't update branches. If you push a new commit to a Renovate branch, for example to fix your code so the tests pass, then Renovate stops all updates of that branch. It is up to you to either finish the job and merge the PR, or rename it and close it so that Renovate can take over again.

I would like to know if there is any configuration to re-enable rebasing these PRs. 

My use case...
Whenever Renovate opens a PR with a dependency update, I would like to run a pipeline on that branch to include a new commit updating my package version to something like CURRENT_VERSION-rc1 (increasing the rc1 to rc2 for example).

If I have multiple dependencies to be updated, it could be problematic as renovate won't rebase the other already opened branches.

Thanks!

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Yes, see https://docs.renovatebot.com/configuration-options/#gitignoredauthors

**Reply by angelbarrera92** [reply]
Seems to be working fine now :)
At least, the warning message is not appearing anymore!
